src/App.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import { Keyboard } from './keyboard';
 
 export const App: React.FC = () => (
   <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
+    <p className="App__message"><Keyboard /></p>
```

Comment:
"Make the App a class component with pressedKey in the state." you don't need separate Keyboard component but make an App class component to do your logic